### DISCUSSION TITLE
Let delays be expressed as range values n1-n2

### DISCUSSION THREAD

**Discussion by a-raccoon** [Nain57/Smart-AutoClicker#890] [General Feature Requests]
### Describe the feature

Instead of a single fixed value, I would like to specify a range of values so a random value in that range is chosen, eg, ```100-10000 ms```, so that a click or wait might be immediate or take as long as 10 seconds.

If possible, I would like lists of values and ranges to be possible, ie, ```100-2000,100-3000,100-4000,8000-10000```, to mix things up and keep clever games from statistically analyzing inputs and delays.

### Describe the usage

The range and list format can be used anywhere a numeric value is inputted, allowing for selecting random values.

### Is it a duplicate?

- [X] There is no duplicates

**Comment by a-raccoon** [comment]
/crickets/

**Comment by lnwncentral** [comment]
Please 👍 the random 5ms changes feel like too little and easily detable, especially when the random click area is also very small.

**Comment by mdp18** [comment]
Second this. I may just create a PR for this.
